### PR TITLE
Added 0xzeze page

### PR DESCRIPTION
**Pull request by 0xzeze**
## Description

I created a my personal page

image

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING
.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

Your ENS/address: 0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3

### CODE REVIEW

File: packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Image from "next/image";
+
+const zeze = () => {
+  return (
+    <>
+      <div className="flex flex-row min-h-full">
```
Comment (phipsae) [inline packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx:7]: You might consider to have a more responsive design using Tailwind CSS to adjust the layout on different screen sizes

File: packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Image from "next/image";
+
+const zeze = () => {
```
Comment (phipsae) [inline packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx:4]: Consider renaming the component to Zeze to follow React's PascalCase convention. 
Also, it's recommended to use the NextPage type from the next package for defining page components in Next.js.

File: packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+              eager to dive in and learn more.
+            </div>
+            <div className="pr-5">
+              <div className="mockup-browser bg-base-300 border">
+                <div className="mockup-browser-toolbar">
+                  <div className="input">My links</div>
+                </div>
+                <div className="bg-base-200 flex items-center justify-between px-12 py-12">
+                  <a className="link link-hover" href="https://github.com/0xzeze" target="_blank">
+                    Github
```
Comment (phipsae) [inline packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx:26]: You could improve the design by incorporating symbols or emojis to make it more visually appealing (github, buidlguidl, element).

**Review by phipsae — CHANGES_REQUESTED** [review]
Thanks for the PR, great job @0xzeze !

I really appreciate the use of the big picture and how you organized the links into a browser tab.

For future PRs, please ensure you **create and check out a new branch** (see contribution guideline, https://gist.github.com/ZakGriffith/69d1eb8baebddd7d370b87a65a7e3ec0).

Below, you'll find a few comments on minor code improvements.

File: packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx (hunk 3 above)
Comment (0xzeze) [reply packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx:26]: Thank you for your reviews. I want to use without emojis

**Comment by 0xzeze** [comment]
Thanks for your reviews. Can you check?

File: packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Image from "next/image";
+
+const page = () => {
```
Comment (derrekcoleman) [inline packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx:4]:
```
Consider renaming the component to Zeze to follow React's PascalCase convention.
Also, it's recommended to use the NextPage type from the next package for defining page components in Next.js.
```

He was recommending you name the component `Zeze`, not `page`. Check out [Rafael's page](https://github.com/BuidlGuidl/batch8.buidlguidl.com/blob/3dfea107c106f7fe31fc3dbdae6401b23b527a41/packages/nextjs/app/builders/0xd1B41bE30F980315b8A6b754754aAa299C7abea2/page.tsx#L7) for an example of how to import and use the `NextPage` type here as well.

**Comment by 0xzeze** [comment]
I changed,thank you

**Comment by derrekcoleman** [comment]
Thanks for renaming it. 

**Please make sure you also implement the `NextPage` as the type of your component**.

**Pull request merged**